pkg/collector/corechecks/snmp/config_metric.go

```diff
@@ -8,8 +8,11 @@ import (
 )
 
 type symbolConfig struct {
-	OID  string `yaml:"OID"`
-	Name string `yaml:"name"`
+	OID          string `yaml:"OID"`
+	Name         string `yaml:"name"`
+	ExtractValue string `yaml:"extract_value"`
```

Comment:
what about `ExtractedValue` ? `Extract` could be considered a function - being an action, a verb

Reply:
Since we use `extract_value` for the yaml field, I wanted to be consistent.